region/brinstar/blue/Blue Brinstar Energy Tank Room.json

```
@@ -1388,12 +1579,22 @@
         ]}
       ],
       "flashSuitChecked": true,
+      "blueSuitChecked": true,
       "note": "After overloading the PLMs by touching the item, go through the crumble and bomb blocks to the top door.",
       "devNote": [
         "The setup for this strat is where the obstacle is cleared.",
         "FIXME: Bombs arent required if the item is there and can be used to overload PLMs. This was just a quick fix to prevent unsoundness."
       ]
     },
+    {
+      "link": [5, 1],
+      "name": "G-Mode Morph",
```

```suggestion
      "name": "Exit G-Mode",
```
This is only needed because of the new [2, 5] strat, right?

---

Right.